app/Http/Controllers/WordController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use App\Models\Word;
+use Illuminate\Http\Request;
+
+class WordController extends Controller
+{
+
+    public function getUserWords(Request $request, int $user_id)
+    {
+        $user = User::find($user_id);
+        if (!$user)
+        {
+            return response("User $user_id not found", 404);
+        }
+
+        $words = Word::where('user_id', $user_id)->get();
+        if (count($words) == 0)
+        {
+            return response("No words found for user $user_id", 404);
+        }
+
+        //placeholder for now; words will always be viewed on a board, right? we wont have stand alone words?
+        $word_list = $words->map(function ($word) {
+            return['text' => $word->text, 'icon' => $word->icon, 'color' => $word->color];
+        })->toArray();
+
+        return response()->json($word_list);
+    }
+
+    public function getWords(Request $request, int $word_id)
+    {
+        $word = Word::find($word_id);
+        if (!$word)
+        {
+            return response("Word $word_id not found", 404);
+        }
+
+        //placeholder
+        return response()->json([
+            'text' => $word->text,
+            'icon' => $word->icon,
+            'color' => $word->color
+        ]);
+    }
+
+    public function createWord(Request $request)
+    {
+        $user = $request->user();
+        #echo $user;
```

Comment:
these `echo` comments can be deleted (also cool tip - Laravel has a builtin function `dd()` that's like `print`/`echo` on steroids)